### ISSUE TITLE
Learning objectives?

### ISSUE THREAD

**Issue by atyre2** [atyre2/NRES450_book#5]
Should each chapter start with a list of learning objectives?

**Comment by GabsPalomo** [comment]
I agree. I'll start thinking about this. Also, I think the questions at the end should tie with the learning objectives somehow. Let me think about the ones for the first chapter and will post them now.

**Comment by atyre2** [comment]
There is also a list of learning objectives in the Index, which comes up as chapter 1, I think. It would be nice to have alignment between these objectives and the chapter objectives.

**Comment by GabsPalomo** [comment]
I just reviewed the general objectives in the Index. What do you think about having General objectives (index) and specific or chapter objectives (for each chapter)? 

This way we can link the general objectives from the book to more specific learning objectives with each chapter.

**Comment by GabsPalomo** [comment]
I found this list of verbs to use for learning objectives according to Bloom's taxonomy. I will keep looking for a list that follows Marzano taxonomy too so we can compare both. 

https://www.clinton.edu/curriculumcommittee/listofmeasurableverbs.cxml

**Comment by GabsPalomo** [comment]
This is definitely the educator in me, and it may be a little too much BUT I think a table like this might help us keep track of the learning objectives per chapter to see which ones we are including and how much weight we are assigning to each one. 

We could include it or not in the Index. Whatever you think is best. This is obviously just a rough draft so we can def do a more "attractive" table. 

![learning objectives per chapter](https://user-images.githubusercontent.com/28967490/50458258-4c4d0000-0927-11e9-94d0-13a1cb03e333.jpg)

**Comment by atyre2** [comment]
Did you do the table in excel or R? I will try to do one in R.
